docs/docs/authenticate/local-auth.mdx

```diff
@@ -3,6 +3,8 @@ title: Local Authentication
 sidebar_position: 1
 ---
 
+# Local authentication (username & password)
```

Comment:
Should this adhere to the new capitalization scheme?

Reply:
see https://github.com/firezone/firezone/pull/1187/files#r1045142538